techniques/fileDistribution/checkGenericFileContent/8.0/tests/simple_content_enforce.json

```diff
@@ -0,0 +1,222 @@
+{
+  "displayName": "Test_checkGenericFileContent_v8.0_simple_content_enforce",
```

Comment:
i based the change on https://github.com/Normation/rudder-techniques/pull/1273/files